src/views/forms/AccountManagementForms/DepositForm.tsx

```diff
@@ -297,13 +310,6 @@ export const DepositForm = ({ onDeposit, onError }: DepositFormProps) => {
       try {
         e.preventDefault();
 
-        if (!signerWagmi) {
```

Comment:
[q] is it safe to remove these? feels like we would still want to check for these things if the wallet type is not phantom, no?

Reply:
ah nevermind, i see it's in an else block